### ISSUE TITLE
Create Issue Dialog -Create Task button should be disabled for users with Viewer role

### ISSUE THREAD

**Issue by sgauruseu** [enonic/app-contentstudio#1968]
1. Create a project and add an user with Viewer role .
2. Login with the user.
3. Click on Create Task menu item.

**BUG:  Create Task button should be disabled**

![image](https://user-images.githubusercontent.com/3728712/84892451-46605700-b0a6-11ea-855a-e99035b4af26.png)

**Comment by sigdestad** [comment]
This is correct - Viewers are users that should only be able to see the published content (as in not access admin console at all). This is used when we cannot use the "everyone" role.

I now see that I regret not calling this role "Audience" (maybe) :-)

**Comment by alansemenov** [comment]
Yes, this is correct but we should disable "Create issue" button

**Comment by sigdestad** [comment]
Yes, but not for this Role - for everyone without the relevant roles
